apps/server/src/modules/management/seed-data/index.ts

```diff
@@ -0,0 +1,35 @@
+import { BaseEntity } from '@shared/domain';
+import { generateFederalStates } from './federalstates';
+import { generateRole } from './roles';
+import { generateSchools } from './schools';
+import { generateSchoolYears } from './schoolyears';
+import { generateSystems } from './systems';
+
+export function generateSeedData(injectEnvVars: (s: string) => string) {
+	// await setupEntities();
+	let collections: { collectionName: string; data: BaseEntity[] }[] = [];
+	// create school related collections
+	const systems = generateSystems(injectEnvVars);
+	const federalStates = generateFederalStates();
+	const years = generateSchoolYears();
+	const schools = generateSchools({ schoolYears: years, systems });
+	// schule,
+
+	// create user related collections
+	const roles = generateRole();
+	// TODO: create user & accounts
```

Comment:
I think with change of the ticket the todos and comments should also be removed.